vectornav/src/main.cpp

```diff
@@ -334,9 +334,9 @@ int run(int argc, char *argv[])
     int async_output_rate;
 
     // Sensor IMURATE (800Hz by default, used to configure device)
-    int SensorImuRate;
-    int minImuRate;
-    int maxImuRate;
+    double SensorImuRate;
+    double minImuRate;
+    double maxImuRate;
```

Comment:
@patapufate I think we should fix this. IIRC the IMU rate is the sampling rate, not the output or publishing rate. The latter is called `async_output_rate`. We should change our diagnostic check parameters to match that e.g.: `max_async_output_rate` and `min_async_output_rate`. It'll be less confusing.